webservices/rest.py

```diff
@@ -525,8 +525,10 @@ def format_totals(self, data, page_data, fields, default_year):
             for record in committee[bucket]:
                 if record != []:
                     details = {}
-                    details['type'] = kind
-                    cycle = int(record['two_yr_period_sk'])
+                    if fields == [] or '*' in fields or 'type' in fields:
+                        details['type'] = kind
+                    if record.has_key('two_yr_period_sk'):
```

Comment:
Just a general note: has_key() has been deprecated and is considered not totally pythonic. SOP is to use in. Not sure if we really care, but wanted to bring it up.